plugins/GSdx/GS.cpp

```diff
@@ -960,6 +961,23 @@ EXPORT_C GSgetTitleInfo2(char* dest, size_t length)
 	strcpy(dest, s.c_str());
 }
 
+EXPORT_C GSgetImageSize(int *image_width, int *image_height)
+{
+	if (!gsopen_done || s_gs == nullptr)
+		return;
+
+	int en0 = s_gs->IsEnabled(0);
+	int en1 = s_gs->IsEnabled(1);
+	if (!en0 && !en1)
+		return;
+
+	// Disregard the disabled circuit. With full boot the BIOS will fill the disabled circuit with its own data.
+	int i = en0 ? 0 : en1 && !en0 ? 1 : -1;
```

Comment:
I think this might cause issues with Time Crisis 2/3 (2 player mode, side by side half width frames IIRC)? Might be better to work with GetInternalResolution() (or whatever it was) that you used in your previous revision.

Side note: `&& !en0` is unnecessary, it's already on the en0 == false path..